scripts/calculate-retroactive-protocol-rewards.ts

```diff
@@ -0,0 +1,62 @@
+const hre = require("hardhat");
+const { DAO_MULTISIG } = require("../params");
+const { ethers } = hre;
+const { provider, BigNumber } = ethers;
+import { currentTime, toUnit, fastForward } from "../test/utils";
+import { network } from "hardhat";
+
+let START_BLOCK = 14293476;
+let END_BLOCK = 14650403;
+let AMOUNT_FOR_GAMMA_UNIV3_SEED = BigNumber.from("180").mul(
+  BigNumber.from(10).pow(18)
+);
+
+async function main() {
+  let tokens = [
+    "0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0", // wsteth
+    "0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599", // wbtc
+    "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48", // usdc
+    "0x7Fc66500c84A76Ad7e9c93437bFc5Ac33E2DDaE9", // aave
```

Comment:
Need to add yvUSDC into the list of tokens

Reply:
added